plugins/commhistory/src/commhistoryplugin.cpp

```diff
@@ -65,6 +65,11 @@ class CommHistoryPlugin::Private
         event.setIsEmergencyCall(handler.isEmergency());
         event.setDirection(handler.isIncoming() ? CommHistory::Event::Inbound
                                                 : CommHistory::Event::Outbound);
+        if (handler.status() == AbstractVoiceCallHandler::STATUS_IGNORED) {
+            event.setIncomingStatus(CommHistory::Event::Ignored);
+        } else if (handler.status() == AbstractVoiceCallHandler::STATUS_REJECTED) {
```

Comment:
Tried adding a phone number to blocked ones and made a call. 

Seems like this is reached with null status, then there's a onVoiceCallStatusChanged() with rejected but that doesn't now update the incoming status.

Reply:
Thanks for testing, I still need to do it myself. I'm a bit busy in the coming days, but should find time to do it properly early next week. I've just added these lines yesterday, to show how things are coming together and PRs are related. But I didn't have time yet to check that everything is properly working.

My early thoughts were that after the commit 2b1afb6dc5eedc91d59a4588b8ed7fdbeb290754, the first status should be INCOMING, REJECTED or IGNORED. But you're right, I forgot that first, status is NULL awaiting for filter decision. I remember it now :/ So I need to implement this a bit differently.